pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -102,27 +102,61 @@ func validatePipelineTaskName(name string) *apis.FieldError {
 func validatePipelineTask(ctx context.Context, t PipelineTask, taskNames sets.String) *apis.FieldError {
```

Comment:
This function is used to validate `tasks` and tasks under `finally` section. Is custom task supported as part of `finally`?

Reply:
Yes it is supported in finally.